### PR TITLE
[Merged by Bors] - chore(Filter/Basic): use `abbrev`

### PR DESCRIPTION
**Pull request by urkud**
Otherwise `coframeMinimalAxioms.toCompleteLattice = instCompleteLatticeFilter` isn't `with_reducible_and_instances rfl`.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by semorrison** [comment]
Could you add the PR description as a comment? Otherwise it's hard to guess why this is an `abbrev`.

**Comment by semorrison** [comment]
bors d+

File: Mathlib/Order/Filter/Basic.lean [hunk 1]
```diff
@@ -785,7 +785,7 @@ instance : DistribLattice (Filter α) :=
           x.sets_of_superset hs inter_subset_right, ht₂, rfl⟩ }
 
 /-- The dual version does not hold! `Filter α` is not a `CompleteDistribLattice`. -/
-def coframeMinimalAxioms : Coframe.MinimalAxioms (Filter α) :=
+abbrev coframeMinimalAxioms : Coframe.MinimalAxioms (Filter α) :=
```
Comment (eric-wieser) [inline Mathlib/Order/Filter/Basic.lean:788]:
```suggestion
-- See note [reducible non-instances]
abbrev coframeMinimalAxioms : Coframe.MinimalAxioms (Filter α) :=
```
or similar

**Comment by urkud** [comment]
bors merge

**Pull request closed without merging**